src/front-end/src/App.js

```diff
@@ -26,15 +26,40 @@ function App() {
             </Menu.Item>
           </Menu>
         </Header>
-        <Content style={{ padding: '0 50px' }}>
+        <Content
+          /* PLEASE READ!
+        Temporary comment for the sake of review: 
+        'minWidth' and 'minHeight' are required so that the footer
+        doesn't cut information off when the browser size is 
+        decreased significantly (i.e the browser is anything but full screen)
+        Disclaimer: minimum sizes are based off of dimensions from 
+        'Extra small devices (portrait phones, less than 576px)'. 
+        */
```

Comment:
Duly noted! :) In the future, if you have comments about your code but that you don't want to actually include in the codebase (here, you'll have to go back and delete it, which is more work), I'd suggest using a comment like the one I'm writing right now, i.e. a GitHub comment rather than a code one.

Reply:
True! Thanks for pointing that out :)